requirements/requirements.txt

```diff
@@ -1,8 +1,9 @@
-ovos-plugin-manager >= 0.0.24a5, < 0.1.0
+ovos-plugin-manager >= 0.0.24, < 0.1.0
 ovos-bus-client>=0.0.5, < 0.1.0
-ovos-utils>=0.0.35a8, < 0.1.0
-ovos_workshop >=0.0.12a44, < 0.1.0
+ovos-utils>=0.0.35, < 0.1.0
+ovos_workshop >=0.0.12, < 0.1.0
 ovos_audio_plugin_simple>=0.0.2a3, < 0.1.0
+# TODO: Stable audio plugin
```

Comment:
This also appears to be a circular dependency as ovos-audio-plugin-simple depends on this package

Reply:
dependency should be removed/moved to extras.txt maybe?

i think its only to ensure a audio backend plugin was available, but not really used by the package